backends/multidimensional_graphite.js

```diff
@@ -1,10 +1,47 @@
-var util = require('util');
-var events = require('events');
+var util = require('util')
+  , events = require('events')
+  , logger = require('../lib/logger')
+  , getHostname = require('os').hostname
+  , getBasename = require('path').basename;
 
 var log = console; // the logger
 var debug;
 
 
+function  createBackendLogger(config, logger) {
```

Comment:
I think that we should push this into the lib/logging and do it by default. At least the debug/info/warn parts.